10up/windows-azure-storage
# Feature/setup cypress for e2e testing

### Description of the Change
This PR adds Cypress for e2e testing, configured and with 2 tests set-up:

1. settings.test - the test adds the admin credentials for a test azure account and uploads a file to verify integration is working
2. admin.test - the test logs in and deactivates and activates the plugin

NOTE: We need to add the test azure account credentials to the GitHub repo but I don't have permissions to do this. The constants should be set as:
MICROSOFT_AZURE_ACCOUNT_NAME
MICROSOFT_AZURE_ACCOUNT_KEY
MICROSOFT_AZURE_CONTAINER


Closes #154

### How to test the Change


1. Pull changes
2. Run npm install
4. Build assets: npm run build
5. Start local environment: npm run env:start
6. Open Cypress for testing supplying test credentials:
`CYPRESS_MICROSOFT_AZURE_ACCOUNT_NAME=[accountname] CYPRESS_MICROSOFT_AZURE_ACCOUNT_KEY=[key] CYPRESS_MICROSOFT_AZURE_CONTAINER=[container] npm run cypress:open`
7. Use Cypress GUI to run tests and verify they are working

### Changelog Entry

> Added - New feature


### Credits


### Checklist:
